cobbler/remote.py

```diff
@@ -3869,7 +3869,7 @@ def get_menu_as_rendered(
         return self.xmlrpc_hacks({})
 
     def get_random_mac(
-        self, virt_type: str = "xenpv", token: Optional[str] = None, **rest: Any
+        self, virt_type: str = "kvm", token: Optional[str] = None, **rest: Any
```

Comment:
Please add a note to the 3.4.0 section in the changelog for the XML-RPC API.

Reply:
I think I figured this out as well.